cmake/versions.json

```diff
@@ -7,7 +7,7 @@
     },
     "Legion": {
       "git_url" : "https://gitlab.com/StanfordLegion/legion.git",
-      "git_tag" : "0ff4a18c08590c28bf3e416581790ceee974665c"
+      "git_tag" : "bc69f142bfffba8781c999dc0637ebbc07b2cd10"
```

Comment:
Can you please bump to 9663e2a0d9f26259dc7ecdac63925c0b5d7408b2, to include a fix for the CI failures seen in https://github.com/nv-legate/cunumeric/pull/932?

Reply:
https://github.com/nv-legate/legate.core/pull/730/commits/f465ea737bda2c2a3eb2c640644daf49734e56a5